applications/DeepSpeed-Chat/training/run_opt-350m.sh

```diff
@@ -1,4 +1,22 @@
 #!/bin/bash
+# --- GPU arch (B200 = SM100) ---
+export TORCH_CUDA_ARCH_LIST="10.0"
```

Comment:
Can this be auto-detected like how [run_opt-13b.sh](https://github.com/LambdaLabsML/DeepSpeedExamples/pull/3/files#diff-53bfcdf7a0215423ccc7ea113642ac0bbbcc1569dcbfc9b40908d217f7dddc61) works?